src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/avoidnoargumentsuperconstructorcall/Example1.java

```diff
@@ -0,0 +1,29 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AvoidNoArgumentSuperConstructorCall"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.avoidnoargumentsuperconstructorcall;
+
+// xdoc section -- start
+class SuperClass {
+  public SuperClass() {}
+
```

Comment:
Let's remove the blank line on Line no.13 also

Reply:
Done :)